numalogic/transforms/_stateless.py

```diff
@@ -91,6 +91,12 @@ def _validate_args(
             if len(lower) != len(upper):
                 raise ValueError("lower and upper should have the same length.")
             lower, upper = np.asarray(lower, dtype=np.float32), np.asarray(upper, dtype=np.float32)
+        elif lower is not None and upper is not None:
+            if type(lower) is not type(upper):
+                if isinstance(lower, Sequence):
+                    upper = np.asarray(upper, dtype=np.float32)
+                else:
+                    lower = np.asarray(lower, dtype=np.float32)
```

Comment:
Did we not decide to make sure that both upper and lower should be consistent types?

Reply:
Correct. But we are still supporting both float and sequence type and either one of them can be null. So for those checks added this